### DISCUSSION TITLE
consider using Kubelinter

### DISCUSSION THREAD

**Discussion by michaelfactor** [fybrik/fybrik#265] [Ideas]
Or alternative tools to ensure the use of secure coding practices

**Comment by roee88** [comment]
@hunchback should be good to use it once we're finished with the m4d helm chart.

**Comment by roee88** [comment]
Workflow for chart validation are available in https://github.com/mesh-for-data/charts#2-add-github-workflow-for-testing-the-chart. We should consider using them as part of our CI but currently it's blocked by the use of deprecated APIs (an open issue)
